Headers/state_met_mod.F90

```diff
@@ -4566,32 +4569,32 @@ SUBROUTINE Get_Metadata_State_Met( am_I_Root, metadataID, Found,   RC,     &
           IF ( isRank  ) Rank  = 2
 
        CASE ( 'FRLAKE' )
-          IF ( isDesc  ) Desc  = 'Fraction of lake'
+          IF ( isDesc  ) Desc  = 'Fraction of lake type in grid box'
           IF ( isUnits ) Units = '1'
           IF ( isRank  ) Rank  = 2
 
        CASE ( 'FRLAND' )
-          IF ( isDesc  ) Desc  = 'Fraction of land'
+          IF ( isDesc  ) Desc  = 'Fraction of land type in grid box'
           IF ( isUnits ) Units = '1'
           IF ( isRank  ) Rank  = 2
 
-       CASE ( 'FRLANDIC' )
-          IF ( isDesc  ) Desc  = 'Fraction of land ice'
+       CASE ( 'FRLANDICE' )
+          IF ( isDesc  ) Desc  = 'Fraction of land ice type in grid box'
           IF ( isUnits ) Units = '1'
           IF ( isRank  ) Rank  = 2
 
        CASE ( 'FROCEAN' )
           IF ( isUnits ) Units = '1'
-          IF ( isDesc  ) Desc  = 'Fraction of ocean'
+          IF ( isDesc  ) Desc  = 'Fraction of ocean type in grid box'
```

Comment:
I suggest removing "type" from the descriptions here. It makes it seem like different categories of lakes, land, ice, and ocean are identified.

Reply:
This is the definition used in the GEOS-FP specification document but it makes sense to remove.

Reply:
Thanks for clarifying